src/lib.rs

```diff
@@ -181,7 +181,7 @@ pub mod schnorr;
 #[cfg(feature = "serde")]
 mod serde_util;
 
-pub use key::{SecretKey, PublicKey, ONE_KEY, KeyPair, XOnlyPublicKey, Parity};
+pub use key::*;
```

Comment:
In 7059192de9e1fb6cf24cb03e8be31a4d6b4f6587:

Will this export non-public things?

Reply:
Actually @TheBlueMatt pointed out to me that there's no way it could, since the top-level module doesn't have visibility into things in `key` that aren't `pub`.

Reply:
Nope. If something is not defined pub, it won't be exported. We doing something similar to context.